purchase_order_fixed_price/models/purchase_order.py

```diff
@@ -0,0 +1,15 @@
+# © 2025 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3.0 (https://www.gnu.org/licenses/lgpl-3.0.html)
+
+from odoo import models, fields, api
+
+class PurchaseOrderLine(models.Model):
```

Comment:
If `purchase.order.line` is the concerned model, file should be named accordingly